pkg/capacityscheduling/elasticquota.go

```diff
@@ -17,12 +17,17 @@ limitations under the License.
 package capacityscheduling
 
 import (
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/kubernetes/pkg/scheduler/framework"
 	"sigs.k8s.io/scheduler-plugins/pkg/util"
 )
 
+const (
+	UpperBoundOfMax = 9223372036854775807
```

Comment:
you can use `math.MaxInt64` here https://pkg.go.dev/math#pkg-constants

Reply:
Thanks. Fixed it.